### ISSUE TITLE
After finishing agent training, inference operation doesn't work normaly.

### ISSUE THREAD

**Issue by KevinRyu** [ChaejungMaeng/Unity_mlagents_Robot_arm#1]
Hi,

I retrained agents and did inference test with recent ml-agents version. 
As a result, agent robot move only one time at start and doesn't move anymore.
I couldn't see any error or warning in unity console but I currently have no idea.
And I modified OnActionReceived function because of deprecated arguments for current ml-agents version.
Do I need real UR robot connection by any chance?
Can you advice about this issue?

Version information:
  ml-agents: 0.26.0,
  ml-agents-envs: 0.26.0,
  Communicator API: 1.5.0,
  PyTorch: 1.10.1+cu113

[INFO] Listening on port 5004. Start training by pressing the Play button in the Unity Editor.
[INFO] Connected to Unity environment with package version 2.1.0-exp.1 and communication version 1.5.0
[INFO] Connected new brain: Robotarm?team=0
[INFO] Hyperparameters for behavior name Robotarm:
        trainer_type:   ppo
        hyperparameters:
          batch_size:   256
          buffer_size:  2560
          learning_rate:        0.0003
          beta: 0.005
          epsilon:      0.2
          lambd:        0.95
          num_epoch:    3
          learning_rate_schedule:       linear
        network_settings:
          normalize:    False
          hidden_units: 256
          num_layers:   3
          vis_encode_type:      simple
          memory:       None
          goal_conditioning_type:       hyper
        reward_signals:
          extrinsic:
            gamma:      0.99
            strength:   1.0
            network_settings:
              normalize:        False
              hidden_units:     128
              num_layers:       2
              vis_encode_type:  simple
              memory:   None
              goal_conditioning_type:   hyper
        init_path:      None
        keep_checkpoints:       5
        checkpoint_interval:    500000
        max_steps:      10000000
        time_horizon:   128
        summary_freq:   50000
        threaded:       False
        self_play:      None
        behavioral_cloning:     None
[
